pkg/api/api.go

```diff
@@ -662,12 +752,12 @@ func equalASCIIFold(s, t string) bool {
 // direct push to the network (default) a pushStamperPutter is returned.
 // returns a function to wait on the errorgroup in case of a pushing stamper putter.
 func (s *Service) newStamperPutter(r *http.Request) (storage.Storer, func() error, error) {
-	batch, err := requestPostageBatchId(r)
+	batch, err := requestPostageBatchId(r) // TODO: extrapolate the headers parsing to the handler level!
```

Comment:
these `TODOs` are meant to be fixed in a future PR?

Reply:
Yes, there is an existing issue for that: #3371